physicsnemo/models/dlwp_healpix_layers/healpix_blocks.py

```diff
@@ -493,7 +492,6 @@ def forward(self, x):
         # return second convnext result
         return self.skip_module2(x1) + self.convblock2(x1)
 
-
 class Multi_SymmetricConvNeXtBlock(th.nn.Module):
```

Comment:
nit: follow naming convention and don't use a `_` in the name

Reply:
Ah dang, I see that has already made it's way into physicsnemo main branch :( nevermind then